dropshot/src/lib.rs

```diff
@@ -507,9 +507,11 @@
  * See the [`RequestInfo`] and [`ResponseInfo`] types for a complete listing
  * of what's available.
  *
- * These probes are implemented via the [`usdt`] crate, and require a nightly
- * toolchain. As such, they're behind the feature flag `"usdt-probes"`. You
- * can build Dropshot with these probes via `cargo +nightly build --features usdt-probes`.
+ * These probes are implemented via the [`usdt`] crate. They require a nightly
+ * toolchain if built on MacOS (which requires the unstable `asm_sym` feature).
+ * Otherwise a stable compiler <= v1.59 is required in order to present the
```

Comment:
I believe you meant "a stable compiler >= 1.59 or a nightly compiler prior to that version". Something along those lines?

Reply:
Yes, silly me.  I'll fix this shortly.